static/src/javascripts/projects/commercial/modules/dfp/dfp-env.ts

```diff
@@ -1,69 +1,86 @@
 import config from '../../../../lib/config';
-import { getUrlVars as _getUrlVars } from '../../../../lib/url';
+import { getUrlVars } from '../../../../lib/url';
 import type { Advert } from './Advert';
 
-const getUrlVars = _getUrlVars as (arg?: string) => Record<string, string>;
-
-interface DfpEnv {
-	renderStartTime: number;
-	adSlotSelector: string;
-	hbImpl: Record<string, boolean>;
-	lazyLoadEnabled: boolean;
-	lazyLoadObserve: boolean;
-	creativeIDs: string[];
-	advertIds: Record<string, number>;
-	advertsToLoad: Advert[];
-	advertsToRefresh: Advert[];
-	adverts: Advert[];
-	shouldLazyLoad: () => boolean;
+interface HbImpl {
```

Comment:
Types are removed in the bundle, so I think we can be a a tad more generous with letter count

```suggestion
interface HeaderBiddingImplementation {
```